mesheryctl/internal/cli/root/system/context.go

```diff
@@ -252,9 +263,14 @@ mesheryctl system context view --all
 * Usage of mesheryctl context view
 # ![context-view-usage](/assets/img/mesheryctl/context-view.png)
 	`,
-	Args:         cobra.MaximumNArgs(1),
+
 	SilenceUsage: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 && (cmd.Flags().Lookup("all").Value.String() == "false" && cmd.Flags().Lookup("context").Value.String() == "") {
+			const errMsg = `Please provide a context name to view.
+Usage: mesheryctl system context view [context name]`
+			return errors.New(utils.SystemContextSubError(fmt.Sprintf("%s\n", errMsg), "context view"))
+		}
```

Comment:
Let's remove this block. `mesheryctl system context view` is a valid command. It automatically picks up the current-context from meshconfig.

Reply:
ok